consensus/service/Cargo.toml

```diff
@@ -26,18 +26,15 @@ mc-ledger-db = { path = "../../ledger/db" }
 mc-ledger-sync = { path = "../../ledger/sync" }
 mc-peers = { path = "../../peers" }
 mc-sgx-report-cache-untrusted = { path = "../../sgx/report-cache/untrusted" }
-mc-sgx-urts = { path = "../../sgx/urts" }
```

Comment:
this might be needed when linking, i'm not sure

Reply:
but actually, if so, it probably appears in the consensus/enclave proxy object's crate?

Reply:
yeah, it's in the `consensus/enclave/Cargo.toml` file which I think is where it should be.